### PR TITLE
feat: create about partner on overview page

### PR DESCRIPTION
**Pull request by ivan-badyulya**
- JIRA -> [create about partner on overview page](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=36&projectKey=PX&modal=detail&selectedIssue=PX-4019&quickFilter=228)

- This PR expects the [PR](https://github.com/artsy/metaphysics/pull/3149) to be merged

Screen Shot 2021-05-12 at 5 20 09 PM

### CODE REVIEW

File: src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}) => {
+  const isEmpty = !profile?.fullBio && !vatNumber && !website
+
+  if (isEmpty) {
+    return null
+  }
+
+  return (
+    <GridColumns mb={140} gridRowGap={2}>
+      <Column span={[12, 6]}>
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx:22]: FWIW you don't have to manually specify the `12` which is there by default if you used the form `span={6}`. Doesn't matter either way.

**Review by dzucconi — APPROVED** [review]
Looks good, minor comment. Merge when ready.

File: src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  partner: { profile, vatNumber, website },
+}) => {
+  const isEmpty = !profile?.fullBio && !vatNumber && !website
+
+  if (isEmpty) {
+    return null
+  }
+
+  return (
+    <GridColumns mb={140} gridRowGap={2}>
```
[the remark this answers is not in the document]
Comment (ivan-badyulya) [reply src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx:21]: Sorry, what do you mean by random value?

Screen Shot 2021-05-12 at 6 39 04 PM

[the remark this answers is not in the document]
Comment (dzucconi) [reply src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx:21]: Sorry, I mean that 140 isn't a sanctioned design system value. Whenever you see something like this it's probably a mistake on design's end. Things should fall within the system.

So like here, `12` (which is 120); is probably the correct value.

[the remark this answers is not in the document]
Comment (ivan-badyulya) [reply src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx:21]: oh, didn't know it, sorry

File: data/schema.graphql [hunk 3]
```diff
@@ -10475,20 +10386,6 @@ type SearchAggregationResults {
...
-  category: String
-  internalID: ID!
-  priceMax: Int
-  priceMin: Int
-  size: String
-}
-
-# A search criteria or errors object
-union SearchCriteriaOrErrorsUnion = Errors | SearchCriteria
-
```
Comment (dzucconi) [inline data/schema.graphql:10491]: Why are these schema updates in this PR? Rebase with master?

Comment (ivan-badyulya) [reply data/schema.graphql]: fixed

**Pull request merged**